ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/css/formcontainerview.css

```diff
@@ -17,6 +17,248 @@
 form:not(.cmp-adaptiveform-container--edit) [data-cmp-visible="false"] {
     display: none !important;
 }
+
+
+form:not(.cmp-adaptiveform-container--edit) [data-cmp-enabled="false"] {
+    opacity: 0.4;
+}
 .cmp-adaptiveform-container--loading{
   
-}
\ No newline at end of file
+}
+
+.cmp-adaptiveform-container [data-cmp-visible="false"] {
```

Comment:
Why do we have the styles defined here? Shouldn't it come from the theme

Reply:
Moved to theme.